data/customers.csv

```diff
@@ -1,3 +1,4 @@
+id,email,street,city,state,zip
```

Comment:
You don't always have the option to directly manipulate data, but this helps make explicit a lot of things.